lib/gtl_formatter.rb

```diff
@@ -27,7 +27,8 @@ class GtlFormatter
   ].freeze
 
   COLUMN_WITH_IMAGE = {
-    'ext_management_system.name' => 'fonticon_or_fileicon'
+    'ext_management_system.name' => 'fonticon_or_fileicon',
+    'resource.name'              => 'fonticon_or_fileicon',
```

Comment:
I mentioned this in https://github.com/ManageIQ/manageiq-ui-classic/pull/8798#discussion_r1202370002, but this change scares me because resource it a generic polymorphic column and COLUMN_WITH_IMAGE is really only meant for EMSes.  I think if we want an EMS reference we should add one to each object that is expecting it, and then the `ext_management_system.name` should Just Work.

Reply:
In this PR I think the only one that falls in that bucket is AuthKeyPair, so it feels like we should add a belongs_to ext_management_system to AuthKeyPair.